flask_session_captcha/logger.py

```diff
@@ -0,0 +1,18 @@
+import logging
+import sys
+
+
+def get_logger(CaptchaName: str = "Flask-Session-Captcha") -> logging.Logger:
+    """ This function return a stdout Logger
+    args:
+        . LoggerName:string: name of the logger
+    """
+    logLevel = logging.INFO
+    logformat = logging.Formatter(f"[{CaptchaName}" + "- %(levelname)s] [%(asctime)s] - %(message)s")
+    logger = logging.getLogger(CaptchaName)
+    logger.setLevel(logLevel)
+    handler = logging.StreamHandler(sys.stdout)
```

Comment:
I'm not that familiar with python's logging, but I think level, format and handlers should be set by the main application, not our plugin.

Reply:
yes in fact but not standard

one of the main key factors for using our logger instead of flask logger is log format, 

the main reason is this way we can format flask-captcha logs differently from flask logs and make it clearer in the terminal for the developer to see

Reply:
In most cases, the user of this plugin will not care about the logs coming from this plugin and this will be seen as unnecessarily noisy/annoying to configure.

Having a specific logger is useful for customization if needed, but we should be using the default logging level, format, handlers etc.